config/application.rb

```diff
@@ -12,7 +12,10 @@ module GovukTopicPagesPrototype
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
-    config.eager_load_paths << Rails.root.join('lib')
+    config.eager_load_paths += %W[
+      #{Rails.root.join('lib')}
+      app/controllers/segment_constraints
```

Comment:
Why did this need to be added? I thought that every folder in `app` was loaded on boot by default?

Reply:
If you remove this you get and uninitialised constant error on the line in the routes.rb file where the FormatConstraint class is used.

https://github.com/alphagov/govuk-topic-pages-prototype/blob/tagged-content-list-page/config/routes.rb#L9